string2int/string2int/string2int.cpp

```diff
@@ -0,0 +1,70 @@
+#include <iostream>
+
+int* str2int(const char* string)
+{
+	//decleration of index representing the position in the string
+	int i = 0;
+	int Negative = 0;
```

Comment:
use bool instead of int; use a prefix for your variable like 'is', e.g., 'isNegativeNumber'